.github/workflows/label_and_milestone_checker.yml

```
@@ -68,3 +68,23 @@ jobs:
             echo "and re-add it when all earlier milestones are completed."
             exit 1
           fi
+
+  check_user_citation_cff:
+    name: Check PR author is in CITATION.cff
+    if: (github.event_name == 'pull_request')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v5
+      - name: Verify PR author exists in CITATION.cff
+        env:
+          PR_AUTHOR: ${{ github.event.pull_request.user.login }}
+        run: |
+          echo "PR author: $PR_AUTHOR"
+          if grep -Ei "^[[:space:]]*alias:[[:space:]]*${PR_AUTHOR}[[:space:]]*$" CITATION.cff; then
+            echo "Found alias '$PR_AUTHOR' in CITATION.cff."
+          else
+            echo "ERROR: GitHub username '$PR_AUTHOR' not found as an alias in CITATION.cff under authors."
+            echo "Please add your GitHub alias to the authors list in CITATION.cff (e.g., 'alias: $PR_AUTHOR')."
```

I think I'd prefer we don't add this explicitly as a burden to contributors. I think we should also offer it as a second PR (yay, 2 PRs!!). Regardless, the responsibility should be on maintainers. Perhaps:
```suggestion
            echo "ERROR: GitHub username '$PR_AUTHOR' not found as an alias in CITATION.cff under authors."
            echo "You are welcome to modify this PR to include your authorship in CITATION.cff."
            echo "Otherwise, a napari core team member will request this information from you."
            echo "Citation format example:
            echo "- given-names: <first name>
            echo "  family-names: <last name>
            echo "  alias: $PR_AUTHOR
            echo "  affiliation: (required)
            echo "  orcid: (optional)
```